deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/wazuh_deployment/wazuh_installation.py

```diff
@@ -5,6 +5,18 @@
 
 
 class WazuhInstallation(ABC):
+    """Install a Wazuh instance set from the configuration file
+
+    Args:
+        wazuh_target (string): Type of the wazuh installation desired (manager or agent).
+        installation_files_path (string): Path where the Wazuh instalation files are located.
+        qa_ctl_configuration (QACTLConfiguration): QACTL configuration.
+    
+    Aributes:
```

Comment:
```suggestion
    Attributes:
```

Reply:
The new modification can be reviewed in the commit:[bba01140837453f908a9dafefa89308ee9a5103e](https://github.com/wazuh/wazuh-qa/commit/bba01140837453f908a9dafefa89308ee9a5103e)